### ISSUE TITLE
authors

### ISSUE THREAD

**Issue by phulm** [schemaorg/schemaorg#3513]
I have just joined. I see no suggestions about how to separate out author's names, i.e. family name, given name, additional name etc. Google's checker throws up a warning if I try to specify them as indicated. Nevertheless, this is what wc3 demands of me in signing in, and in some places I want to specify the family name first, in other places the given name. Has any work been done on this? Grateful for any info.

**Comment by MaxwellDAnderson** [comment]
Hi @phulm, if I understand your question, you are trying to add family name, given name and additional name to an author. To do this, you need to make sure you add the Person type to the author property. So your syntax would look something like this:

```
<html>
  <body itemscope itemtype="http://schema.org/CreativeWork">
    <p itemprop="author" itemscope itemtype="http://schema.org/Person">
      <span itemprop="givenName">John</span>
      <span itemprop="additionalName">Q.</span>
      <span itemprop="familyName">Public</span>
    </p>
  </body>
</html>
```
This does not throw any errors on Schema's validation tool, available here: [https://validator.schema.org/](https://validator.schema.org/). 

Hope this helps!
